src/rcc.rs

```diff
@@ -457,7 +483,41 @@ impl CFGR {
         self
     }
 
-    /// Enables the Plli2S clock source.
+    /// Enables the PLLSAI clock source.
+    pub fn use_pllsai(mut self) -> Self {
+        self.use_pllsai = true;
+        self
+    }
+
+    /// Sets the PLLSAIN multiplication factor for PLLSAI.
+    ///
+    /// # Panics
+    ///
+    /// Panics if the multiplication factor isn't between 50 and 432.
```

Comment:
Possible slight clarification:
```suggestion
    /// Panics if the multiplication factor isn't between 50 and 432 (inclusive).
```